components/hidden-text/src/test.js

```diff
@@ -32,6 +32,13 @@ describe('hidden text', () => {
     });
   });
 
+  it('should be closed by default', () => {
+    const output = mount(<HiddenText />);
```

Comment:
You can use shallow here instead, it's a little more performant and people seem to harp on about it being best practice, I did some tests locally as I am pretty new to enzyme but this works nicely;

```
it('should be closed by default', () => {
  const output = shallow(<HiddenText />);
  expect(output.prop('open')).toEqual(undefined);
});
```

Reply:
@Loque- as discussed, discovered that shallow is not sufficient to access the HTML element with the "open" attribute.

Is this now ok to merge?

Reply:
Thanks for the looking at that Joe!